GoogleForCreators/web-stories-wp
# Fixed canvas focus

## Context

This fixed the canvas focus and re-enables relevant tests.

## Relevant Technical Choices

* Missing `forwardRef` on new component between layers and DOM element.

## User-facing changes

Auto-focus works again

## Testing Instructions

This PR can be tested by following these steps:

1. Add a new text element using the + button in the element library tab bar
1. Press <kbd>enter</kbd> to edit text
1. Observe the edit mode is working and text is select

## Checklist




---

Fixes #9781
